### ISSUE TITLE
UI issues

### ISSUE THREAD

**Issue by floringogianu** [EliverLara/Ant#9]
Love the theme, a couple of issues I'm seeing:
- [ ] Maximize window control (the yellow one) not showing.
- [ ] Back, Forward (<>) window controls are white on a very light gray with very low contrast and therefore are basically invisible.
- [ ] Radio buttons are not rendered at all (checkboxes are).
 
![image](https://user-images.githubusercontent.com/1670348/30785912-8fc0b738-a176-11e7-9ad6-6d5472981724.png)

**Comment by EliverLara** [comment]
Hi @floringogianu thanks for reporting this, could you provide information about your OS? please.

**Comment by floringogianu** [comment]
Let me know if you require anything else.

**OS**:      `Ubuntu 17.04`
**DE**:      `Budgie 10.4-0ubuntu0.1`
**WM**:     `Mutter 3.24.2-0ubuntu0.1`
**libgtk3**: `3.22.11-0ubuntu3`

**Comment by floringogianu** [comment]
Some updates. I can confirm that when opening a gtk3 application with a different scale factor for regular resolutions (eg. `env GDK_SCALE=0.5 GDK_DPI_SCALE=0.5 CLUTTER_SCALE=0.5 nautilus &|`) the maximize control shows up just fine. 

Here are the errors and warnings when opening nautilus without any environment variables, I see some related to the gtk theme:

```
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:267:12: not a number

(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:267:12: Expected a string.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:437:70: Using one color stop with linear-gradient() is deprecated.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:450:70: Using one color stop with linear-gradient() is deprecated.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:463:70: Using one color stop with linear-gradient() is deprecated.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:473:70: Using one color stop with linear-gradient() is deprecated.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:497:70: Using one color stop with linear-gradient() is deprecated.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:514:70: Using one color stop with linear-gradient() is deprecated.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:527:70: Using one color stop with linear-gradient() is deprecated.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:537:70: Using one color stop with linear-gradient() is deprecated.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:559:72: Using one color stop with linear-gradient() is deprecated.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:570:72: Using one color stop with linear-gradient() is deprecated.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:594:72: Using one color stop with linear-gradient() is deprecated.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:603:12: not a number
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:603:12: Expected a string.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:727:73: Using one color stop with linear-gradient() is deprecated.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:740:58: Using one color stop with linear-gradient() is deprecated.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:751:72: Using one color stop with linear-gradient() is deprecated.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:760:73: Using one color stop with linear-gradient() is deprecated.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:768:12: not a number
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:768:12: Expected a string.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:892:72: Using one color stop with linear-gradient() is deprecated.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:905:58: Using one color stop with linear-gradient() is deprecated.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:916:72: Using one color stop with linear-gradient() is deprecated.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:925:72: Using one color stop with linear-gradient() is deprecated.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:993:12: not a number
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:993:12: Expected a string.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:1001:12: not a number
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:1001:12: Expected a string.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:1652:14: not a number
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:1652:14: Expected a string.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:1949:54: Using one color stop with linear-gradient() is deprecated.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:1956:59: Using one color stop with linear-gradient() is deprecated.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:2436:16: not a number
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:2436:16: Expected a string.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:2441:16: not a number
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:2441:16: Expected a string.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:2488:16: not a number
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:2488:16: Expected a string.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:2493:16: not a number
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:2493:16: Expected a string.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:2565:16: not a number
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:2565:16: Expected a string.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:2672:18: not a number
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:2672:18: Expected a string.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:2717:72: Using one color stop with linear-gradient() is deprecated.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:3614:74: Using one color stop with linear-gradient() is deprecated.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:3624:74: Using one color stop with linear-gradient() is deprecated.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:3634:74: Using one color stop with linear-gradient() is deprecated.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:3643:74: Using one color stop with linear-gradient() is deprecated.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:4332:70: Using one color stop with linear-gradient() is deprecated.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:4341:70: Using one color stop with linear-gradient() is deprecated.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:4350:70: Using one color stop with linear-gradient() is deprecated.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:4358:70: Using one color stop with linear-gradient() is deprecated.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:4399:9: Not using units is deprecated. Assuming 'px'.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:4399:9: Expected a string.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:4781:8: not a number
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:4781:17: Using Pango syntax for the font: style property is deprecated; please use CSS syntax
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:4882:16: not a number
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:4882:16: Expected a string.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:4904:21: The '-gtk-image-effect' property has been renamed to '-gtk-icon-effect'
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:5142:47: The :inconsistent pseudo-class is deprecated. Use :indeterminate instead.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:5261:16: not a number
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:5261:16: Expected a string.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:5377:52: The :prelight pseudo-class is deprecated. Use :hover instead.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:5468:10: Not using units is deprecated. Assuming 'px'.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:5468:10: Expected a string.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:5740:12: not a number
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:5740:12: Expected a string.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:5747:14: not a number
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:5747:14: Expected a string.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:5759:16: not a number
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:5759:16: Expected a string.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:5764:14: not a number
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:5764:14: Expected a string.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:5778:12: not a number
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:5778:12: Expected a string.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:5825:70: Using one color stop with linear-gradient() is deprecated.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:5834:70: Using one color stop with linear-gradient() is deprecated.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:5843:70: Using one color stop with linear-gradient() is deprecated.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:5851:12: Expected a string.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:5854:16: not a number
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:5854:16: Expected a string.
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:5862:12: not a number
(nautilus:10405): Gtk-WARNING **: Theme parsing error: gtk.css:5862:12: Expected a string.

sys:1: PyGIWarning: Nautilus was imported without specifying a version first. Use gi.require_version('Nautilus', '3.0') before import to ensure that the right version gets loaded.
Initializing nautilus-dropbox 2015.10.28

** (nautilus:10405): CRITICAL **: nautilus_menu_provider_get_background_items: assertion 'NAUTILUS_IS_MENU_PROVIDER (provider)' failed

** (nautilus:10405): CRITICAL **: nautilus_menu_provider_get_background_items: assertion 'NAUTILUS_IS_MENU_PROVIDER (provider)' failed
Nautilus-Share-Message: Called "net usershare info" but it failed: Failed to execute child process “net” (No such file or directory)

** (nautilus:10405): CRITICAL **: nautilus_menu_provider_get_background_items: assertion 'NAUTILUS_IS_MENU_PROVIDER (provider)' failed

** (nautilus:10405): CRITICAL **: nautilus_menu_provider_get_background_items: assertion 'NAUTILUS_IS_MENU_PROVIDER (provider)' failed
```

**Comment by EliverLara** [comment]
Thanks for the info.

**About window controls (close,minimize, maximize) and checks/radios**

I tested the theme under ubuntu 17.10 (with budgie desktop and Gtk 3.22.11), and as you can see in the screenshot below everything seems to be ok.

![captura de pantalla de 2017-09-27 15-52-10](https://user-images.githubusercontent.com/9838260/30937711-37736426-a39d-11e7-9d17-a40842a40825.png)

**About Back, Forward (<>) window controls**

The very light gray color is present when they are disabled, if you navigate through your folders these will become black.


In conclusion I do not know what is happening here, maybe you're right when you talk about missing assets for 4k resolutions,  but sadly i have no way to check that.

**Comment by oddPulse** [comment]
I'm on zorin os 12.2 core. the UI most often keeps crashing when you open an app or when you switch a window by going to activities overview then selecting the app you wanna switch to. using

**Comment by Alkaris** [comment]
On Linux Mint 17.3 I can't select different theme colors, there's only white theme colors, and window borders look like they're using Adwaita theme style.

**Comment by EliverLara** [comment]
Fixed in last commit.
